addons/viewports/CfgVehicles.hpp

```diff
@@ -0,0 +1,169 @@
+class CfgVehicles {
+    class B_MBT_01_base_F;
+    class B_MBT_01_cannon_F: B_MBT_01_base_F { // Merkel
+        class ADDON {
+            class SLD_backLeftUpper {
+                type = "screen";
+                camLocation[] = {0,0,0.05};
```

Comment:
Would be good to also allow memory point in here.
Then future made vehicles can just directly provide the camera memory points inside the model

Reply:
I would second this, if this could use memory points, it would make adding this to new vehicles pretty easy and would be a very nice addition.